connectors/sync_job_runner.py

```diff
@@ -192,6 +195,15 @@ async def execute(self):
             await self._sync_done(sync_status=JobStatus.SUSPENDED)
         except ConnectorJobCanceledError:
             await self._sync_done(sync_status=JobStatus.CANCELED)
+        except ElasticAuthorizationException as e:
+            index_name = (
+                self.connector.acl_index_name
+                if job_type == JobType.ACCESS_CONTROL
+                else self.connector.index_name
+            )
```

Comment:
You don't need this. You can always use `self.sync_job.index_name`:

https://github.com/elastic/connectors/blob/2d5df6a3ac11a6b6124886c980f406ed7ce1d052/connectors/protocol/connectors.py#L1002-L1007

Reply:
Oh neat! I've changed it to use this. Thank you.